db/schema.rb

```diff
@@ -12,14 +12,19 @@
 #
```

Comment:
@ChristopherJMiller can we pull in that schema fixer gem

Reply:
@torresed please investigate into adding this gem before we accept the PR https://github.com/jakeonrails/fix-db-schema-conflicts

Reply:
Gem` fix-db-schema-conflicts` is incompatible with Ruby 3.0.